universal-login-sdk/lib/core/services/validations/RecipientValidator.ts

```diff
@@ -1,28 +1,37 @@
 import {Validator, TransferErrors} from './Validator';
 import {utils} from 'ethers';
 import {isValidEnsName, TransferDetails} from '@universal-login/commons';
+import UniversalLoginSDK from '../../../api/sdk';
+import {AddressZero} from 'ethers/constants';
 
-function isProperAddress(recipient: string): boolean {
+const isProperAddress = (recipient: string): boolean => {
```

Comment:
Merge with isProperAddress in commons

Reply:
isProperAddress in commons looks like this:
```ts
export const isProperAddress = (address: string) => {
  return !!address.match(/^0x[0-9A-Fa-f]{40}$/);
};
```
And here we use ethers to define if address is valid.

Reply:
I can update isProperAddress in commons in next PR